http/vertx-web-client/src/test/resources/application.properties

```diff
@@ -0,0 +1,10 @@
+# Configuration file
+quarkus.test.native-image-profile=test
+quarkus.http.port=8081
```

Comment:
Do we need this properties fie? Also the property `quarkus.http.port` should not be in.

Reply:
Well, I guess it's mostly needed for timeouts and to show which properties do we replace.

Reply:
Property removed